azure-pipelines.yml

```diff
@@ -141,6 +141,9 @@ stages:
               /p:ManifestBuildBranch=$(Build.SourceBranchName)
               /p:ManifestBuildNumber=$(Build.BuildNumber)
               /p:VisualStudioDropName=Products/dotnet/aspnetcore-tooling/$(Build.SourceBranchName)/$(Build.BuildNumber)
+        
+        - ${{ if eq(variables['System.TeamProject'], 'public') }}:
+          - enableRichCodeNavigation: true
```

Comment:
Why is this enabled here too❔

Reply:
For now I've added it to test and get a gauge for how much time Rich Nav will add to your CI if we chose to add it to the main CI. I only intend to merge _either_ the changes to the CI build _or_ the new Rich Nav CI